lib/pci_caps.c

```diff
@@ -181,19 +181,19 @@ cap_write_msix(vfu_ctx_t *vfu_ctx, struct pci_cap *cap, char *buf,
      */
 
     if (msix->mxc.fm != new_msix.mxc.fm) {
+        msix->mxc.fm = new_msix.mxc.fm;
         if (new_msix.mxc.fm) {
```

Comment:
nit, could you make this refer to msix->mcx.fm now?

Reply:
Sure!